scoreboard/src/routes.py

```diff
@@ -264,20 +264,54 @@ def kerplunk():
         print(f"ignoring metrics exception: {e}")
 
     errors = [
-        (400, "Bad Request: Frog tried to cross before the light turned green."),
-        (401, "Unauthorized: Toad with no tokens detected."),
-        (403, "Forbidden: Hopper not cleared for this lane."),
-        (404, "Not Found: Frog missed the lily pad."),
-        (418, "I'm a Teapot: And also a flattened amphibian."),
-        (429, "Too Many Requests: The crosswalk is jammed with turtles."),
-        (500, "Internal Server Error: Frogger hit by a memory bus."),
-        (502, "Bad Gateway: Log drift caused route misalignment."),
-        (503, "Service Unavailable: Arcade machine took a coffee break."),
-        (504, "Gateway Timeout: Frog paused mid-hop. Unwise."),
+        (502, "Bad Ad Gateway: Log drift caused route misalignment."),
+        (503, "Ad Service Unavailable: Arcade machine took a coffee break."),
     ]
     code, message = random.choice(errors)
+    current_span.set_attribute("error.message", message)
+    current_span.set_attribute("version", version)
```

Comment:
make sure our traces have attributes we can use in tag spotlight and trace analyzer and so on